mammoth/options.py

```diff
@@ -30,15 +30,14 @@ def _get_line(line):
 p.Heading2 => h2:fresh
 p.Heading3 => h3:fresh
 p.Heading4 => h4:fresh
+p.Heading5 => h5:fresh
+p.Heading6 => h6:fresh
 p[style-name='Heading 1'] => h1:fresh
 p[style-name='Heading 2'] => h2:fresh
 p[style-name='Heading 3'] => h3:fresh
 p[style-name='Heading 4'] => h4:fresh
-p[style-name='heading 1'] => h1:fresh
-p[style-name='heading 2'] => h2:fresh
-p[style-name='heading 3'] => h3:fresh
-p[style-name='heading 4'] => h4:fresh
-p[style-name='heading 4'] => h4:fresh
+p[style-name='heading 5'] => h5:fresh
+p[style-name='heading 6'] => h6:fresh
```

Comment:
The different cases are there to handle different applications (LibreOffice and Word). At the moment, style names are case insensitive (but weren't in the past), but in case the behaviour is ever changed back, then I think it's useful to have the two separate cases. Might be neater to collapse the cases and a comment, but if so, then that's a different issue and should be handled separately.